internal/pgdump/pgdump.go

```diff
@@ -59,8 +59,8 @@ type Output struct {
 // path. It can be provided a zero-value Targets to just generate the output paths.
 func Outputs(dir string, targets Targets) []Output {
 	return []Output{{
-		Output: filepath.Join(dir, "primary.sql"),
-		Target: targets.Primary,
+		Output: filepath.Join(dir, "pgsql.sql"),
+		Target: targets.Pgsql,
```

Comment:
It's been a long time since I've been here, but are there any other process docs we need to update that may reference this DB by its old name?